src/main/java/uk/gov/hmcts/cmc/claimstore/models/ccj/RepaymentPlan.java

```diff
@@ -1,6 +1,7 @@
 package uk.gov.hmcts.cmc.claimstore.models.ccj;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.fasterxml.jackson.annotation.JsonProperty'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AUselessImportCheck)